packages/gatsby-theme-bodiless/create-node.js

```diff
@@ -61,6 +66,30 @@ const findFilesystemNode = ({ node, getNode }) => {
   return fsNode;
 };
 
+/**
+ * Get current git repo info.
+ *
+ * @returns Promise<{
+ *  repo: string,
+ *  hash: string,
+ *  branch: string,
+ * }>
+ */
+const createGitInfo = async () => {
+  const gitDir = await findGitFolder();
+  const projectRoot = pathUtil.dirname(gitDir);
+  const remotes = await git.listRemotes({ fs, dir: projectRoot });
+  const origin = remotes.filter(v => v.remote === 'origin');
+  const repo = (origin.length === 1) ? origin[0].url : '';
```

Comment:
Using `Array.find` would probably make these lines clearer:

```suggestion
  const origin = remotes.find(v => v.remote === 'origin');
  const repo = origin?.url ?? '';
```